base/regex.jl

```diff
@@ -214,16 +215,12 @@ function match(re::Regex, str::Union{SubString{String}, String}, idx::Integer, a
     n = div(length(ovec),2) - 1
     mat = SubString(str, ovec[1]+1, prevind(str, ovec[2]+1))
     cap = Union{Void,SubString{String}}[ovec[2i+1] == PCRE.UNSET ? nothing :
-                                        SubString(str, ovec[2i+1]+1,
-                                                  prevind(str, ovec[2i+2]+1)) for i=1:n]
+            SubString(str, ovec[2i+1]+1, prevind(str, ovec[2i+2]+1)) for i=1:n]
```

Comment:
Incorrect alignment.

Reply:
This was the alignment used before my PR. What alignment should be correct. I have decreased in my local file the alignment to one tab so `S` in `SubString` is exactly below `=` in the above line.

Reply:
I'd say the alignment from your PR was OK, at least it made it clear that these where inside the comprehension. (Anyway, if you didn't align everything inside the comprehension, I think it should use only four spaces, not eight; we never use tabs, so one character = one space, that's a simple rule.)

Reply:
ok - reverting to the original alignment from the PR (by tab I meant 4 spaces)